pkg/egressgateway/manager_privileged_test.go

```diff
@@ -180,18 +191,22 @@ func newEgressPolicyConfig(policyName string, labels map[string]string, destinat
 	}
 }
 
-func newEndpoint(name, ip string, labels map[string]string) k8sTypes.CiliumEndpoint {
+func newEndpoint(name, ip string, eplb map[string]string) k8sTypes.CiliumEndpoint {
 	epLabels := []string{}
-	for k, v := range labels {
+	for k, v := range eplb {
+		//epLabels = append(epLabels, fmt.Sprintf("k8s:%s=%s", k, v))
+		// Not sure about this, probably not need the "k8s:"" prefix
```

Comment:
I think it's fine to keep the prefix and then use it in `Map2Labels()`

Reply:
Done. Thx.